client/src/style/scss/base.scss

```diff
@@ -999,10 +999,7 @@ a.action-button {
 // A content button is a button in content items
 
 .content-buttons {
-    .btn:hover,
-    .btn:active,
-    .btn:focus,
-    .btn:focus-visible {
+    .btn {
         background-color: transparent !important;
         border-color: transparent !important;
         box-shadow: none !important;
```

Comment:
Just discovered that this removes active highlighting. Seems to be introduced by an earlier PR. Removing it restores highlighting for me, without reintroducing style discrepancies. Could you check if this is also the case on Safari?
```suggestion
```